tests/js/spec/utils/profiling/renderers/cursorRenderer.spec.tsx

```diff
@@ -0,0 +1,45 @@
+import {vec2} from 'gl-matrix';
+
+import {LightFlamegraphTheme} from 'sentry/utils/profiling/flamegraph/FlamegraphTheme';
+import {Rect, Transform} from 'sentry/utils/profiling/gl/utils';
+import {CursorRenderer} from 'sentry/utils/profiling/renderers/cursorRenderer';
+
+describe('CursorRenderer', () => {
+  it('renders cursor in center screen', () => {
+    const context = {
+      beginPath: jest.fn(),
+      moveTo: jest.fn(),
+      lineTo: jest.fn(),
+      strokeStyle: undefined,
+      stroke: jest.fn(),
+    };
+    const canvasMock = {
+      getContext: jest.fn().mockReturnValue(context),
+    } as unknown as HTMLCanvasElement;
```

Comment:
Would it be worthwhile to create a helper to create mock canvases? Perhaps something under `TestStubs`?

Reply:
Yeah, we could, but tbh it's such a small mock that I prefer to just inline it. We can have a look after we port over all the renderers, could easily be done.